src/main/java/com/googlecode/lanterna/gui2/AbstractListBox.java

```diff
@@ -173,7 +173,7 @@ public synchronized Result handleKeyStroke(KeyStroke keyStroke) {
      * @return index of a item that was clicked on with {@link MouseAction}
      */
     protected int getIndexByMouseAction(MouseAction click) {
-        return click.getPosition().getRow() - getGlobalPosition().getRow();
+        return Math.min(click.getPosition().getRow() - getGlobalPosition().getRow(), items.size() -1);
```

Comment:
When a list has the screen real estate to present more items than the list actually has in it, there will be blank space at the bottom rows.
This change makes it so that clicking in that blank space selects the last item rather than throwing an exception.